Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -362,7 +362,29 @@ public class RepositoryPackageContainer: BasePackageContainer, CustomStringConve
         // Compute the map of known versions.
         //
         // FIXME: Move this utility to a more stable location.
-        let knownVersions = Git.convertTagsToVersionMap(repository.tags)
+        let knownVersionsWithDuplicates = Git.convertTagsToVersionMap(repository.tags)
+        
+        let knownVersions = knownVersionsWithDuplicates.mapValues { (tags) -> String in
+            if tags.count == 1 {
+                return tags[0]
+            }
+            else if tags.count == 2 {
+                do {
+                    let normalRevision = try repository.resolveRevision(tag: tags[0])
+                    let vPrefixedRevision = try repository.resolveRevision(tag: tags[1])
```

Comment:
Since we are not really emitting a warning right now, we shouldn't do this work. These are somewhat costly operations.